### PR TITLE
Major change for the server.js

### PR DESCRIPTION
**Pull request by thomassuedbroecker**
image

basic-understanding-own_modules

### CODE REVIEW

File: own_modules/customCloudant.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  			raw: false,
+  	  	auth: { username: cloudantUser, password: cloudantPassword }
+    });
+  	db = connection.database(dbName);
+  	db.exists(function(err,exists){
+  				if(err){
+  					console.log('Error accessing db' + err );
+  					cb(err,null);
+  				}else if(exists){
+  						console.log('db ' + dbName + ' exists');
```
Comment (rhocheck) [inline own_modules/customCloudant.js:32]: meh, some ugly tabs :)

File: own_modules/customCloudant.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  		else{
+  			cb(null,'views created');
+  		}
+  	});
+  }
+
+  this.createDoc = function(doc, cb){
+  	db.save(doc, function (err, res) {
+        if(err){
+  				cb(err,null);
```
Comment (rhocheck) [inline own_modules/customCloudant.js:74]: tabs won't ever look nice on a diff

File: own_modules/customCloudant.js (hunk 1 above)
Comment (rmey) [reply own_modules/customCloudant.js:32]: agree tabs need to be cleaned up

**Comment by rmey** [comment]
I don't want to have web app in that Microservice. The purpose of the app is simple feeding database from IoT events either as local App, Docker or in Bluemix as multi-instance Container Group or CloudFoundry App. A web app is useful for sensor visualization, which should connect to the database.

**Pull request closed without merging**